packages/backend/src/tokens/verify.ts

```diff
@@ -31,6 +31,18 @@ export async function verifyToken(
       // Fetch JWKS from Backend API using the key
       key = await loadClerkJWKFromRemote({ ...options, kid });
     } else {
+      if (!options.secretKey || !options.jwtKey) {
```

Comment:
🔧 Based on the error message i think the check should be `!options.secretKey && !options.jwtKey`.  Also do you think there is a case where this if will not be executed? If any of the secretKey or jwtKey exists, the if / else if above will be executed. Based on that i think that condition should be removed and we should update the existing error message or keep it as is.

Reply:
@dimkl i think i should just to this, because we already handle the missing JWT key case.

I'm still using the same reason as before, in order not to mess with any caller that only expects the `JWKFailedToResolve` reason, but i've updated the action and message to be more descriptive, and give hint on that it needs to be fixed.
```suggestion
 if (!options.secretKey) {
        return {
          errors: [
            new TokenVerificationError({
              action: TokenVerificationErrorAction.SetClerkSecretKey,
              message: 'Secret Key is missing. Could not load remote JWK.',
              reason: TokenVerificationErrorReason.JWKFailedToResolve,
            }),
          ],
        };
      }
```